.github/workflows/create_gui_java.yml

```diff
@@ -1,56 +1,59 @@
 name: Create gui-java Template
 
 on:
-  push:
-    paths-ignore:
-      - "tests/**"
-      - "docs/**"
-  pull_request:
-    paths-ignore:
-      - "tests/**"
-      - "docs/**"
+    push:
+        paths-ignore:
+            - "tests/**"
+            - "docs/**"
+    pull_request:
+        paths-ignore:
+            - "tests/**"
+            - "docs/**"
 
 jobs:
-  build:
-
-    runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[ci skip]')"
-    strategy:
-      matrix:
-        python: [3.7, 3.8]
-
-    steps:
-      - uses: actions/checkout@v2
-        name: Check out source-code repository
-
-      - name: Setup Python
-        uses: actions/setup-python@v2.2.2
-        with:
-          python-version: ${{ matrix.python }}
-
-      - name: Build Cookietemple
-        run: |
-          python setup.py clean --all install
-
-      - name: Create gui-java Template
-        run: |
-          echo -e "\033[B\033[B\n\nHomer\nhomer.simpson@hotmail.com\nhomergithub\nnExplodingSpringfield\ndescription\n1.0.0\n\nburnsorga\nn" | cookietemple create
-
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
-        with:
-          java-version: 11
-
-      - name: Compile Package
-        run: |
-             cd explodingspringfield
-             make compile
-
-      - name: Install doc dependencies
-        run: |
-             pip install -r docs/requirements.txt
-
-      - name: Build docs
-        run: |
-             cd docs
-             make html
+    build:
+        runs-on: ubuntu-latest
+        if: "!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[ci skip]')"
+        strategy:
+            matrix:
+                python: [3.8, 3.9]
+
+        steps:
+            - uses: actions/checkout@v2
+              name: Check out source-code repository
+
+            - name: Setup Python
+              uses: actions/setup-python@v2.2.2
+              with:
+                  python-version: ${{ matrix.python }}
+
+            - name: Install Poetry
+              run: |
```

Comment:
So I am now trying to always skip the `|` for one-liners, but add it if there are two lines or more.